esphome/components/time/real_time_clock.cpp

```diff
@@ -23,6 +23,13 @@ namespace time {
 static const char *const TAG = "time";
 
 RealTimeClock::RealTimeClock() = default;
+
+void RealTimeClock::dump_config() {
+#ifdef USE_TIME_TIMEZONE
+  ESP_LOGCONFIG(TAG, "  Timezone: '%s'", this->timezone_.c_str());
```

Comment:
```suggestion
  ESP_LOGCONFIG(TAG, "Timezone: '%s'", this->timezone_.c_str());
```

Since its a different tag